src/manage.py

```diff
@@ -92,7 +93,7 @@ def test():
         COV.report()
         COV.html_report()
         COV.stop()
-        COV.erase()
+        subprocess.run(['coveralls'])
```

Comment:
Why don't you just add this to the tests script?

Reply:
Commanded added to the scripts file.